semgrep-core/src/core/ast/Bloom_filter.ml

```diff
@@ -18,13 +19,17 @@ module B = Bloomf
 (* TODO *)
 type elt = string
 
+type filter = Bloom of elt B.t | Set of elt Set.t ref
```

Comment:
Can you explain why do we need a ref for Set?

Reply:
You can set that the API for adding elements to the bloom/set is imperative, and
B.t internally use references, so for Set we need too.